src/Stratis.Bitcoin.IntegrationTests/BlockStore/ProofOfWorkSpendingSpecification.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
 using Xunit;
+// ReSharper disable ArrangeThisQualifier
```

Comment:
should we do this in the shared settings file for integration tests?

Reply:
yes good idea